conf/pacific/rados/6-node-cluster-4-clients.yaml

```diff
@@ -54,46 +54,30 @@ globals:
           openstack: RHEL-8.7.0-x86_64-ga-latest
           ibmc: rhel-87-server-released
         networks:
-          - provider_net_cci_8
+          - provider_net_cci_13
         role:
           - client
       node8:
         image-name:
           openstack: RHEL-8.7.0-x86_64-ga-latest
           ibmc: rhel-87-server-released
         networks:
-          - provider_net_cci_8
+          - provider_net_cci_13
         role:
           - client
       node9:
         image-name:
           openstack: RHEL-8.7.0-x86_64-ga-latest
           ibmc: rhel-87-server-released
         networks:
-          - provider_net_cci_9
+          - provider_net_cci_10
         role:
           - client
       node10:
         image-name:
           openstack: RHEL-7.9-x86_64-ga-latest
           ibmc: rhel-server-79-x86-64-kvm
```

Comment:
If all the other clients are on RHEL8, any reason as to why this is on RHEL7?

Reply:
This is on pacific. On quincy and reef, it's RHEL 9.

I can change this as well to RHEL 8 if needed.

Reply:
Have Updated to latest RHEL8 image on all clients.